README.md

```diff
@@ -107,15 +107,15 @@ over a WebSocket in the form of plain JSON (i.e., the data is sent as a string,
 as opposed to being serialised and transferred as binary data).
 
 The MHA Python server fulfills two purposes: it *serves* the web applications
-over HTTP (thus enabling remote viewing), and it acts as a *bridge* between the
-web application and an MHA instance.  For the latter, it receives commands from
-the web app over the aforementioned WebSocket, and communicates with an MHA
-instance over TCP via MHA's simple network protocol (see the file
-`MHAConnection.py` for the details).  Note that the TCP connection is made
-on-demand, in order to accommodate other MHA clients, since the MHA only
-supports a single TCP connection.  However, this does not appear to add an
-appreciable amount of overhead.  Both the HTTP and WebSocket components are
-implemented with the [Tornado](http://www.tornadoweb.org/) Python library.
+over HTTP, and it acts as a *bridge* between the web application and an MHA
+instance.  For the latter, it receives commands from the web app over the
+aforementioned WebSocket, and communicates with an MHA instance over TCP via
+MHA's simple network protocol (see the file `MHAConnection.py` for the
+details).  Note that the TCP connection is made on-demand, in order to
+accommodate other MHA clients, since the MHA only supports a single TCP
+connection.  However, this does not appear to add an appreciable amount of
```

Comment:
> MHA only supports a single TCP connection.
Since openMHA release 4.6.2 mha accepts multiple connections.

Reply:
Yes, I almost commented on that but silently pushed my WIP branch that uses a single connection instead.